src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,42 @@
 
 class Animal {
   // write your code here
+  static alive = [];
```

Comment:
The `alive` array is used to track all instances of `Animal`. However, there is no method to remove an animal from this array when it dies, except in the `Carnivore` class. Consider adding a method in the `Animal` class to handle this logic.